Billing.API/Startup.cs

```diff
@@ -18,7 +20,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddDopplerSecurity();
-            services.AddTaxInfoProvider();
+            services.AddInvoice();
```

Comment:
Very minor, and opinable, of course:

```suggestion
            services.AddInvoiceService();
```

```suggestion
            services.AddInvoicesService();
```

```suggestion
            services.AddInvoicesProvider();
```

Reply:
Changed...
Three times? lols